pkg/builder/cluster/convert/testdata/gcs-archive-sources.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+sources:
+- name: sname
+  targetPath: "workspace/foo"
+  gcs:
+    type: Archive
+    location: gs://bucket/my/stuff.tar.gz
+- gcs:
+    type: Archive
+    location: gs://bucket/my/stuff.tar.gz
+  targetPath: "workspace/foo"
+steps:
+- name: helloworld
+  image: gcr.io/google-containers/busybox
```

Comment:
If this doesn't actually have to be a real image, since it's not actually going to run anything, this could just be:

```
steps:
- image: hello
```

Reply:
Sure, I will update this file. I think there are examples in testdata that declare `gcr.io/google-containers/busybox` as image. 
ref: https://github.com/knative/build/blob/master/pkg/builder/cluster/convert/testdata/gcs-archive.yaml#L20